integrations/github/README.md

```diff
@@ -0,0 +1,147 @@
+# Technical Spec For Nango GitHub Pull Requests And Commits Sync
```

Comment:
Adding this file to the repo isn't necessary and is more for internal purposes. The relevant information should go into the sync specific markdown document, so `commits.md` for example